games-puzzle/pingus/pingus-0.7.6.recipe

```diff
@@ -0,0 +1,65 @@
+SUMMARY="Free Lemmings clone"
+DESCRIPTION="Pingus is a free Lemmings(tm) clone for GNU/Linux, Win32 and \
+other operating systems supported by SDL. It is covered under the GPL."
+HOMEPAGE="https://pingus.seul.org"
+COPYRIGHT="1998-2011 Ingo Ruhnke"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/Pingus/downloads/blob/master/pingus-$portVersion.tar.bz2?raw=true"
```

Comment:
It would be better to use the project download based on a tag:
https://github.com/Pingus/pingus/archive/v0.7.6.tar.gz